resolwe_bio/processes/reads_processing/cutadapt.yml

```diff
@@ -165,8 +175,8 @@
       cutadapt \
         {% if headcrop %}-u {{headcrop}}{% endif %} \
         {% if crop %}-u -{{crop}}{% endif %} \
-        {% if leading %}-q {{leading}}{% endif %} \
-        {% if trailing %}-q {{trailing}},0{% endif %} \
+        {% if trailing %}-q {{ trailing }}{% endif %} \
```

Comment:
@JenkoB @mzganec @JureZmrzlikar For the numerical inputs, the existance of the variable should be tested using `{% if var is defined %}`. If the user inputs 0, the `{% if var %}` evaluates to `False`, which skips the use of the parameter flag in the process altogether. This is an issue in (too) many places, not just in the process being worked on here.